packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -77,17 +78,97 @@ enum SystemUiOverlay {
   bottom,
 }
 
+/// Specifies a dark vs light preference for certain system chrome options.
+enum SystemChromeTheme {
```

Comment:
We've been using `FooTheme` to name inherited widgets (typically configured with `FooData`) that configure visuals for descendant `Foo` widgets. 

Instead of defining a new enum here, could we use [Brightness](https://docs.flutter.io/flutter/material/Brightness-class.html)

Reply:
Having services provide Brightness sounds sensible to me.